src/Modules/AuthenticationModule/components/ChangePass/ChangePass.tsx

```diff
@@ -0,0 +1,173 @@
+import { Visibility, VisibilityOff } from "@mui/icons-material";
+import {
+  Alert,
+  Box,
+  Button,
+  FormControl,
+  IconButton,
+  InputAdornment,
+  InputLabel,
+  OutlinedInput,
+  Typography,
+} from "@mui/material";
+import { useEffect, useRef, useState } from "react";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { ChangePasswordProps } from "../../../../interfaces/interface";
+import { userRequest } from "../../../../utils/request";
+import { getErrorMessage } from "../../../../utils/error";
+import { useToast } from "../../../Context/ToastContext";
+import { useNavigate } from "react-router-dom";
+
+export default function ChangePass() {
+  const [oldPass, setOldPass] = useState(false);
+  const [newPass, setNewPass] = useState(false);
+  const [confirmPass, setConfirmPass] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const navigate = useNavigate()
+  const timeoutRef = useRef<number | null>(null);
+  const { showToast } = useToast();
+
+  const {
+    register,
+    handleSubmit,
+    getValues,
+    formState: { errors },
+  } = useForm<ChangePasswordProps>();
+
+  const onSubmit: SubmitHandler<ChangePasswordProps> = async (data) => {
+    setLoading(true);
+    try {
+      const res = await userRequest.post("/admin/users/change-password", data);
+      showToast("success", res.data.message);
+      timeoutRef.current = setTimeout(()=>{
+        navigate("/login")
```

Comment:
I don't recommend having this timer, as it makes  our application feel slow. We also shouldn't redirect the user to the login page after changing the password as the user is already signed in.

Reply:
-Fixed the empty active field
-booking page was empty as there is no bookings added ,but if we add bookings it will work
-fixed redirect